### PR TITLE
Updated SM Cascade event to be more lore friendly.

### PR DESCRIPTION
**Pull request by Xoxeyos**
## About The Pull Request

This PR makes the announcement for the SM cascade much more curt, and makes it more lore friendly.

## Why It's Good For The Game

We aren't a station, there is no escape shuttle, and everyone in the galaxy will know who to be pissed at, for their lost benefits.

## Changelog
:cl: Xoxeyos
tweak: SM Cascade event is now more lore friendly.
/:cl:

### CODE REVIEW

**Comment by Xoxeyos** [comment]
I'm open to suggestions for changes regarding the dialogue.

File: code/game/gamemodes/endgame/supermatter_cascade/universe.dm [hunk 1]
```diff
@@ -58,15 +58,11 @@ var/global/universe_has_ended = 0
 
 	spawn(rand(30,60) SECONDS)
 		var/txt = {"
-There's been a galaxy-wide electromagnetic pulse.  All of our systems are heavily damaged and many personnel are dead or dying. We are seeing increasing indications of the universe itself beginning to unravel.
+AUTOMATED ALERT: Attention [station_name()], this is a high alert broadcast to all ships from the central communication hub of the Hanza Trade Union, a catastrophe has happened on the ship [station_name()], information regarding the incident is classified.
```
Comment (Firefox13) [inline code/game/gamemodes/endgame/supermatter_cascade/universe.dm:61]:
```suggestion
AUTOMATED ALERT: Attention [station_name()], this is a high alert broadcast to all ships from the central communication hub of the Hansa Trade Union, a catastrophe has happened on the ship [station_name()], information regarding the incident is classified.
```

File: code/game/gamemodes/endgame/supermatter_cascade/universe.dm [hunk 2]
```diff
@@ -58,15 +58,11 @@ var/global/universe_has_ended = 0
...
 
-[station_name()], you are the only facility nearby a bluespace rift, which is near your research outpost. You are hereby directed to enter the rift using all means necessary, quite possibly as the last of your species alive.
-
-You have five minutes before the universe collapses. Good l\[\[###!!!-
+We highly suggest, that all corporate owned, and free ships within listening range depart into Bluespace. Until the incident ends, all pay-rolls will be frozen, and employees aboard currently employed HTU ships will have benefits cut.
```
Comment (Firefox13) [inline code/game/gamemodes/endgame/supermatter_cascade/universe.dm:63]: Eris is indeed one of the Hansa registered ships but it's still owned by captain and crew is paid by them and not by HTU, so it's probably best to remove the pay-roll part.

Comment (Firefox13) [reply code/game/gamemodes/endgame/supermatter_cascade/universe.dm:63]: I understand that it's probably an automated message sent to all ships but that just sounds weird

File: code/game/gamemodes/endgame/supermatter_cascade/universe.dm (hunk 1 above)
Comment (Xoxeyos) [reply code/game/gamemodes/endgame/supermatter_cascade/universe.dm]: Thank you!

File: code/game/gamemodes/endgame/supermatter_cascade/universe.dm (hunk 2 above)
Comment (Xoxeyos) [reply code/game/gamemodes/endgame/supermatter_cascade/universe.dm]: I've added in an addition that independent ships are free from their payroll being cut, however I believe the players will have to have the brains to understand this, I do not them babied.

**Review by TheShown911 — APPROVED** [review]

**Pull request merged**